apps/src/lab2/views/components/Instructions/ResourcePanel/index.tsx

```diff
@@ -169,11 +165,8 @@ const ResourcePanel: React.FC<ResourcePanelProps> = ({
   // Tooltip should disappear quickly.
   const hideTooltipDelayMs = 10;
 
-  // Temporary read-only occurs when running/validating in a workspace that wasn't permanently read-only at mount.
-  const isTemporarilyReadOnly =
-    !isPermanentlyReadOnlyRef.current &&
-    isReadOnly &&
-    (isRunning || isValidating);
+  // Temporary read-only occurs when running/validating in a workspace that isn't permanently read-only.
```

Comment:
nit: could update comment to not reference running/validating since that logic could potentially change in the future. We could say something like "if we are not permanently read only but are currently in a read only state, we are temporarily read only".